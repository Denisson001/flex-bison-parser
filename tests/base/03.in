6
4 -1 10 100 -66 8