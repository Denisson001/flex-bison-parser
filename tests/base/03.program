{
    a = 12;
    b = a;
    b = a + b;

    str_s = "123";
    string_s = str_s;

    string_s[0] = "2";

    print(a);
    print(b);
    print(string_s);
}