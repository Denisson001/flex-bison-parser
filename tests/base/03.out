12
24
223
